### PR TITLE
Fix master

### PR DESCRIPTION
**Pull request by eapolinario**
Signed-off-by: Eduardo Apolinario <eapolinario@users.noreply.github.com>

# TL;DR
Bump flytekit in integration tests and regenerate requirements

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
Integration tests pin the flytekit version here. We regenerated the requirements files with the newly released version of flytekit.

__
## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

**Review by wild-endeavor — APPROVED** [review]

**Comment by eapolinario** [comment]
The notebook format is described in https://nbformat.readthedocs.io/en/latest/format_description.html. There's a library called `nbformat` that provides a better interface for dealing with notebooks, but it doesn't do anything useful for metadata, which is what this PR is about, so we'll keep the approach.
